### ISSUE TITLE
Godef doesn't build now that 9fans has moved to github?

### ISSUE THREAD

**Issue by skastel** [rogpeppe/godef#4]
I was setting up godef today on a new laptop and upon running ```go get github.com/rogpeppe/godef``` I received an error:
```
[user@localhost]$ go get code.google.com/p/rog-go/exp/cmd/godef
package code.google.com/p/rog-go/exp/cmd/godef
        imports 9fans.net/go/acme: unrecognized import path "9fans.net/go/acme"
```

seeing what appeared to be the issue, I changed the imports to in acme.go to:
```
"github.com/9fans/go/acme"
```
And that started to help, but then all the [9fans] (https://www.github.com/9fans) packages are still importing things like ```9fans.net/go/plan9``` so the same problem kept arising. Eventually I changed all the ```9fans.net/*``` imports to ```github.com/9fans/*``` and go get command completed successfully.

I may just be building this incorrectly or my setup could be broken, so let me know if that's the case! I'm running go 1.4.2 built from source on Fedora 21 64-bit. Let me know if there's any other info I can provide that will help.

**Comment by txthinking** [comment]
too

**Comment by medvednikov** [comment]
Same problem here.

**Comment by johnjelinek** [comment]
:+1:

**Comment by mengzhuo** [comment]
same issue here
@medvednikov 
can you give me an workaround for Vundle install ?

**Comment by nyankosama** [comment]
yeah, the same problem came to me too.

**Comment by rogpeppe** [comment]
I need to set up a notification on godef issues! Buried amongst a pile of github emails, I haven't seen this before. I will vendor its dependencies forthwith. Thanks for the report.

**Comment by razic** [comment]
@rogpeppe fwiw i think it's fixed now

**Comment by rogpeppe** [comment]
Fixed by #35
